tools/cli/commands.go

```diff
@@ -967,6 +967,24 @@ func describeWorkflowHelper(c *cli.Context, wid, rid string) {
 		o = convertDescribeWorkflowExecutionResponse(resp)
 	}
 	prettyPrintJSONObject(o)
+	if resp.WorkflowExecutionInfo.AutoResetPoints != nil && len(resp.WorkflowExecutionInfo.AutoResetPoints.Points) > 0 {
```

Comment:
Can we add a flag, reason is https://github.com/uber/cadence/issues/1728 we may want most command be able to print parseable json. 
This table looks an add-on, I would not want to see it in `wf desc`, but would like to only see this when `wf desc -preset`

Reply:
Will do.